src/ssl.c

```diff
@@ -14457,7 +14457,14 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
                 idx += (int)iov[i].iov_len;
             }
 
+           /* myBuffer may not initialized fully, but the sending length will be */
+#if defined(__GNUC__) && __GNUC__ > 5
```

Comment:
Since we have this multiple times, make macros for them.
One for PRAGMA_GCC_IGNORE(str) and one PRAGMA_GCC_POP.

Reply:
The ignores may be dependent on the version. Here I used version 5 as the gate. @SparkiDev For now I'll add a macro with that fixed at 5, but add a comment that we may need to split it based on version.

Reply:
You can change the name to PRAGMA_GCC5_* if you like.